src/features/network/pages/network-page.tsx

```diff
@@ -26,5 +28,21 @@ export function NetworkPage({ children }: Props) {
     }
   }, [hash, pathname, search, navigate, selectedNetwork, isWildcardNetworkRoute])
 
+  useEffect(() => {
```

Comment:
I'm wondering if we should move this outside of a react component. My reasoning is that the logic to read the params is outside of a React component => the logic to delete them should be outside too.

Reply:
We could extract into a hook, but it needs to needs to still run as an effect, as it uses react router to do the delete.